lib/linkerd2-opencensus/src/metrics.rs

```diff
@@ -0,0 +1,71 @@
+use linkerd2_metrics::{metrics, Counter, FmtMetrics};
+use std::fmt;
+use std::sync::{Arc, Mutex};
+use tracing::error;
+
+metrics! {
+    span_export_streams: Counter { "Total count of opened span export streams" },
+    span_export_requests: Counter { "Total count of span export request messages" },
+    span_exports: Counter { "Total count of spans exported" }
```

Comment:
Sorry to catch this right at the end, but can we namespace this?  "span" is not an obvious prefix to me. WDYT about "opencensus_span_.."